src/systems/ec/implementations/react/packages/checkbox/stories/Index.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import demo from '@ecl/ec-specs-checkbox/demo/data';
+import Checkbox from '../Checkbox';
+
+storiesOf('Forms/Checkbox', module)
+  .add('default', () => <Checkbox {...demo.default} />)
+  .add('disabled', () => <Checkbox {...demo.disabled} />)
+  .add('invalid', () => (
+    <Checkbox {...demo.invalid} onClick={e => e.preventDefault()} />
```

Comment:
Why is there a onClick here?
Invalid checkbox should be usable. If the user can't check it, he can't solve the error

Reply:
Mmhh..but we don't provide this behaviour. We only define the styles and the markup associated, the preventDefault there was just to avoid the inconsistent result of a checked checkbox with errors still in place, if you started playing with it, it was not demoing the behavior of a form element with errors. 
So maybe this should be a generic onChange callback that removes the error classes and elements from all the components if they contain any.
This checkbox in react has been defined as an uncontrolled component, i am now handling this behavior with an onChange callback for this component only, don't know whether this is the right approach.